### PR TITLE
add website team

### PR DESCRIPTION
**Pull request by varodrig**
updating the https://github.com/open-gitops/project/blob/main/TEAMS.md
adding the website team.

### CODE REVIEW

**Review by niklasmtj — APPROVED** [review]

**Comment by niklasmtj** [comment]
Hey @varodrig would you mind verifying your commit so the DCO bot is happy? You can find the instructions in the [action's summary](https://github.com/open-gitops/project/pull/193/checks?check_run_id=28478758115). Otherwise this looks great!

File: TEAMS.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Please [reach out in any of the available ways](https://github.com/open-gitops/.
...
 | marketing/events | active | [purpose](https://github.com/open-gitops/project/blob/main/GOVERNANCE.md#team-members) (also see [#151](https://github.com/open-gitops/project/issues/151)) | <ul><li>[@christianh814](https://github.com/christianh814)</li><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@roberthstrand](https://github.com/roberthstrand)</li><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@staceypotter](https://github.com/staceypotter)</li></ul> |
+| website | active | [purpose](https://github.com/open-gitops/project/blob/main/GOVERNANCE.md#team-members) (also see [#151](https://github.com/open-gitops/project/issues/155)) | <ul><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@JaimeMagiera](https://github.com/JaimeMagiera)</li><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@varodrig](https://github.com/varodrig)</li></ul> |
```
Comment (scottrigby) [inline TEAMS.md:27]: requesting two changes:
- no need to copy the previous info column format (tbh can't remember why we did that)
- please remove Jaime (he said he [can't commit](https://cloud-native.slack.com/archives/C026D3RE7FB/p1722890051417599?thread_ts=1722887707.550829&cid=C026D3RE7FB) right now)

```suggestion
| website | active | [current discussion](https://github.com/open-gitops/project/issues/155) | <ul><li>[@scottrigby](https://github.com/scottrigby)</li><li>[@niklasmtj](https://github.com/niklasmtj)</li><li>[@varodrig](https://github.com/varodrig)</li></ul> |
```

**Review by scottrigby — CHANGES_REQUESTED** [review]
Thanks for this PR! Just a few changes please

File: TEAMS.md (hunk 1 above)
Comment (varodrig) [reply TEAMS.md:27]: done! thank you for reviewing

**Review by scottrigby — APPROVED** [review]
I comitted those suggestions. Thanks @varodrig, let's do this! 😄

**Pull request merged**